EnoLauncher/Program.cs

```diff
@@ -146,23 +146,47 @@ public async Task LaunchCheckerTask(CheckerTask task)
 
         static void Main()
         {
-            var serviceProvider = new ServiceCollection()
-                .AddScoped<IEnoDatabase, EnoDatabase.EnoDatabase>()
-                .AddDbContextPool<EnoDatabaseContext>(options =>
+            const string mutexId = @"Global\EnoLauncher";
+            bool createdNew;
+            using (var mutex = new Mutex(false, mutexId, out createdNew))

```

Comment:
Can we do an early return to reduce the total indentation?

Reply:
no, because that wont work nice with the try-finally statement...

Reply:
i decided against making a new function here like in the Engine because the code there was just a few lines